mmengine/infer/infer.py

```diff
@@ -139,7 +139,8 @@ def __init__(self,
                  model: Union[ModelType, str, None] = None,
                  weights: Optional[str] = None,
                  device: Optional[str] = None,
-                 scope: Optional[str] = None) -> None:
+                 scope: Optional[str] = None,
+                 show_progress: bool = True) -> None:
```

Comment:
Please update the docstring

Reply:
> Please update the docstring

Sorry, I forgot to modify the relevant text. Now I added a description of the parameters in the file